widgetry/src/event_ctx.rs

```diff
@@ -168,6 +168,7 @@ impl<'a> EventCtx<'a> {
                 self.style
                     .loading_tips
                     .clone()
+                    .default_fg(Color::WHITE)
```

Comment:
Unrelated fix: because the loading mostly doesn't leverage the theming stuff, and just has a bunch of hardcoded colors, it also needs to have a hardcoded text color, otherwise it's not legible in the new day mode.

One day we should probably theme the loading screen too.